clients/src/main/resources/common/message/ShareFetchRequest.json

```diff
@@ -0,0 +1,72 @@
+  // explicitly enabled.
+  "latestVersionUnstable": true,
+  "fields": [
+    { "name": "GroupId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null", "entityType": "groupId",
+      "about": "null if not provided or if it didn't change since the last fetch; the group identifier otherwise." },
+    { "name": "MemberId", "type": "string", "versions": "0+", "nullableVersions": "0+",
+      "about": "The member ID." },
+    { "name": "ShareSessionEpoch", "type": "int32", "versions": "0+",
+      "about": "The current share session epoch: 0 to open a share session; -1 to close it; otherwise increments for consecutive requests." },
+    { "name": "MaxWaitMs", "type": "int32", "versions": "0+",
+      "about": "The maximum time in milliseconds to wait for the response." },
+    { "name": "MinBytes", "type": "int32", "versions": "0+",
+      "about": "The minimum bytes to accumulate in the response." },
+    { "name": "MaxBytes", "type": "int32", "versions": "0+", "default": "0x7fffffff", "ignorable": true,
+      "about": "The maximum bytes to fetch. See KIP-74 for cases where this limit may not be honored." },
+    { "name": "Topics", "type": "[]FetchTopic", "versions": "0+",
+      "about": "The topics to fetch.", "fields": [
+      { "name": "TopicId", "type": "uuid", "versions": "0+", "ignorable": true, "about": "The unique topic ID"},
+      { "name": "Partitions", "type": "[]FetchPartition", "versions": "0+",
+        "about": "The partitions to fetch.", "fields": [
+        { "name": "PartitionIndex", "type": "int32", "versions": "0+",
+          "about": "The partition index." },
+        { "name": "CurrentLeaderEpoch", "type": "int32", "versions": "0+", "default": "-1", "ignorable": true,
+          "about": "The current leader epoch of the partition." },
+        { "name": "PartitionMaxBytes", "type": "int32", "versions": "0+",
+          "about": "The maximum bytes to fetch from this partition. See KIP-74 for cases where this limit may not be honored." },
+        { "name": "AcknowledgementBatches", "type": "[]AcknowledgementBatch", "versions": "0+",
+          "about": "Record batches to acknowledge.", "fields": [
+          { "name": "StartOffset", "type": "int64", "versions": "0+",
+            "about": "Start offset of batch of records to acknowledge."},
+          { "name": "LastOffset", "type": "int64", "versions": "0+",
+            "about": "Last offset (inclusive) of batch of records to acknowledge."},
+          { "name": "GapOffsets", "type": "[]int64", "versions": "0+",
+            "about": "Array of offsets in this range which do not correspond to records."},
+          { "name": "AcknowledgeType", "type": "int8", "versions": "0+", "default": "0",
+            "about": "The type of acknowledgement - 0:Accept,1:Release,2:Reject."}
+        ]}
+      ]}
+    ]},
+    { "name": "ForgottenTopicsData", "type": "[]ForgottenTopic", "versions": "0+", "ignorable": false,
```

Comment:
I don't think KIP talked about this or I missed. Please if you can help regarding the purpose of `ForgottenTopicsData`.

Reply:
Yes, indeed. It's missing. The purpose is the same as KIP-227 to remove topic-partitions from a share session.